scripts/common.sh

```
@@ -127,7 +128,26 @@ status_json() {
+    local players_cache cache_mtime now
+    players_cache="${PZ_STATUS_RCON_CACHE:-${PZ_CACHEDIR}/status-rcon-players.cache}"
+    cache_mtime="$(stat -c %Y "$players_cache" 2>/dev/null || printf 0)"
+    now="$(date +%s)"
+    if [ $((now - cache_mtime)) -lt 15 ]; then
+      players_raw="$(cat "$players_cache" 2>/dev/null || true)"
+    else
+      players_raw="$(rcon_cmd_quick players 2>/dev/null || true)"
+      if mkdir -p "$(dirname "$players_cache")" 2>/dev/null; then
+        printf '%s' "$players_raw" > "${players_cache}.pztmp" 2>/dev/null \
+          && chmod 600 "${players_cache}.pztmp" 2>/dev/null \
+          && mv "${players_cache}.pztmp" "$players_cache" 2>/dev/null \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)